inventory/models.py

```diff
@@ -1,10 +1,17 @@
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 class Customer(models.Model):
+    class Type(models.TextChoices):
+        RETAIL = "RE", _("Retail")
+        WHOLESALE = "WH", _("Wholesale")
+
```

Comment:
Can we add a new `partner` customer type in order to identify customers that we don't necessarily sell to or buy from?